include/acutest.h

```diff
@@ -1313,6 +1327,9 @@ test_cmdline_callback__(int id, const char* arg)
             test_worker__ = 1;
             test_worker_index__ = atoi(arg);
             break;
+        case 'x':
+            test_xml_output__ = fopen(arg, "wb");
```

Comment:
Please add an error handling here if the `fopen()` fails. `fprintf(stderr, "some sane error message\n"); exit(2);` is good enough.

Reply:
Oops, yuo were faster.